### ISSUE TITLE
Error while Deploying

### ISSUE THREAD

**Issue by SairajMN** [NethermindEth/zksync-remix-plugin#134]
Error in Deploying the SimpleStorage.sol with zkSync pulgin, the complilation of code went well but the Deploy button is not visible (I'm unable to deploy it).
![Screenshot 2024-07-03 204859](https://github.com/NethermindEth/zksync-remix-plugin/assets/112056445/2ed6ddf7-9c6f-41de-8491-ca857c4d0e80)

**Comment by varex83** [comment]
Hello @SairajMN ! 

It seems like you put your `SimpleStorage.sol` file into the wrong directory. Our plugin reproduces the standard structure of hardhat project, so all of your contracts should be in the `contracts` directory, or they won't be compiled. While our team is working on providing better interface, you can just create `contracts` directory and put your `SimpleStorage.sol` there, and try compiling it again.

Let me know if it helped you!

**Comment by SairajMN** [comment]
I did that too.

**Comment by SairajMN** [comment]
This time it shows deploy button but when I hit deploy it's shows errors in logs and I'm not getting it.

 Should I just have to clear all the files and folders in Remix and create new contracts folder and execute it?

**Comment by varex83** [comment]
can you send us errors you get, please?

**Comment by SairajMN** [comment]
Here is error  : 

![Screenshot 2024-07-05 201944](https://github.com/NethermindEth/zksync-remix-plugin/assets/112056445/856250c6-2d77-4c3c-a9b7-d48ec051c2ff)

it show something like this
```
Failed to compile:
Error HH404: File @chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol, imported from contracts/PriceConverter.sol, not found.
HardhatError: HH404: File @chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol, imported from contracts/PriceConverter.sol, not found.
```
in Terminal

here is my code of SimpleStorage.sol
```solidity

// SPDX-License-Identifier: MIT

pragma solidity 0.8.24;

contract SimpleStorage {
    uint256 myFavoriteNumber;

    struct Person {
        uint256 favoriteNumber;
        string name;
    }
    Person[] public listOfPeople;

    mapping(string => uint256) public nameToFavoriteNumber;

    function store(uint256 _favoriteNumber) public {
        myFavoriteNumber = _favoriteNumber;
    }

    function retrieve() public view returns (uint256) {
        return myFavoriteNumber;
    }

    function addPerson(string memory _name, uint256 _favoriteNumber) public {
        listOfPeople.push(Person(_favoriteNumber, _name));
        nameToFavoriteNumber[_name] = _favoriteNumber;
    }
}
```
And in created a new folder as contracts

![Screenshot 2024-07-05 201826](https://github.com/NethermindEth/zksync-remix-plugin/assets/112056445/a2be7c64-d99f-412d-b55c-265307f41ee2)

**Comment by SairajMN** [comment]
I don't know if it's related to the another contract FundMe.sol, if it is then here is code of that 
``` solidity
// SPDX-License-Identifier: MIT
pragma solidity 0.8.24;

import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol";


library PriceConverter {
    function getPrice() internal view returns (uint256) {
        AggregatorV3Interface priceFeed = AggregatorV3Interface(
            0xfEefF7c3fB57d18C5C6Cdd71e45D2D0b4F9377bF
        );
        (, int256 answer, , , ) = priceFeed.latestRoundData();
        return uint256(answer * 10000000000);
    }

    function getConversionRate(
        uint256 ethAmount
    ) internal view returns (uint256) {
        uint256 ethPrice = getPrice();
        uint256 ethAmountInUsd = (ethPrice * ethAmount) / 1000000000000000000;
        return ethAmountInUsd;
    }
}


error NotOwner();

contract FundMe {
    using PriceConverter for uint256;

    mapping(address => uint256) public addressToAmountFunded;
    address[] public funders;

    address public /* immutable */ i_owner;
    uint256 public constant MINIMUM_USD = 5 * 10 ** 18;

    constructor() {
        i_owner = msg.sender;
    }

    function fund() public payable {
        require(msg.value.getConversionRate() >= MINIMUM_USD, "You need to spend more ETH!");
        addressToAmountFunded[msg.sender] += msg.value;
        funders.push(msg.sender);
    }

    function getVersion() public view returns (uint256) {
        AggregatorV3Interface priceFeed = AggregatorV3Interface(0xfEefF7c3fB57d18C5C6Cdd71e45D2D0b4F9377bF);
        return priceFeed.version();
    }

    modifier onlyOwner() {
        // require(msg.sender == owner);
        if (msg.sender != i_owner) revert NotOwner();
        _;
    }

    function withdraw() public onlyOwner {
        for (uint256 funderIndex = 0; funderIndex < funders.length; funderIndex++) {
            address funder = funders[funderIndex];
            addressToAmountFunded[funder] = 0;
        }
        funders = new address[](0);
        // // transfer
        // payable(msg.sender).transfer(address(this).balance);

        // // send
        // bool sendSuccess = payable(msg.sender).send(address(this).balance);
        // require(sendSuccess, "Send failed");

        // call
        (bool callSuccess,) = payable(msg.sender).call{value: address(this).balance}("");
        require(callSuccess, "Call failed");
    }
    // Explainer from: https://solidity-by-example.org/fallback/
    // Ether is sent to contract
    //      is msg.data empty?
    //          /   \
    //         yes  no
    //         /     \
    //    receive()?  fallback()
    //     /   \
    //   yes   no
    //  /        \
    //receive()  fallback()

    fallback() external payable {
        fund();
    }

    receive() external payable {
        fund();
    }
}
```

it shows the same error if i run FundMe.sol

**Comment by satyambnsal** [comment]
Fixed as part of [0.4.2](https://github.com/NethermindEth/zksync-remix-plugin/releases/tag/v0.4.2) release.
